test/test_va_api_fixture.cpp

```diff
@@ -498,9 +522,13 @@ VAAPIFixtureSharedDisplay::VAAPIFixtureSharedDisplay() : VAAPIFixture() { }
 
 void VAAPIFixtureSharedDisplay::SetUpTestSuite()
 {
+#if defined(_WIN32)
+    {
+        s_vaDisplay = getWin32Display(NULL);
+#else
     if (s_drmHandle < 0) {
         s_vaDisplay = getDrmDisplay(s_drmHandle);
-
+#endif
```

Comment:
1. Why you did not modify `TearDownTestSuite`? It seems you are missing `vaTerminate` for windows.
2. I wish you would minimize complex macro constructions. For example, you don't exclude `s_drmHandle` and you can use that even on Windows:
```
void VAAPIFixtureSharedDisplay::SetUpTestSuite()
{
    if (s_drmHandle < 0) {
#if defined(_WIN32)
        s_vaDisplay = getWin32Display(NULL);
#else
        s_vaDisplay = getDrmDisplay(s_drmHandle);
#endif
        int majorVersion, minorVersion;
        s_initStatus = vaInitialize(s_vaDisplay, &majorVersion, &minorVersion);
    }
}

void VAAPIFixtureSharedDisplay::TearDownTestSuite()
{
    if (s_vaDisplay) {
        if (s_initStatus == VA_STATUS_SUCCESS) {
            vaTerminate(s_vaDisplay);
            s_vaDisplay = nullptr;
        } else {
            s_initStatus = VA_STATUS_SUCCESS;
        }
    }
    if (s_drmHandle >= 0) {
        close(s_drmHandle);
        s_drmHandle = -1;
    }
}
```